s2a_options.go

```diff
@@ -103,6 +103,8 @@ type ClientOptions struct {
 	//     ensureProcessSessionTickets.Wait()
 	//   }
 	EnsureProcessSessionTickets *sync.WaitGroup
+	// V2 is a flag that enables S2Av2.
+	V2 bool
```

Comment:
nit: Please call this `EnableV2` and change the comment to "If true, enables the use of S2Av2."

Similarly below.